src/realm/sync/noinst/migration_store.hpp

```diff
@@ -40,25 +40,27 @@ class MigrationStore : public std::enable_shared_from_this<MigrationStore> {
     MigrationStore& operator=(const MigrationStore&) = delete;
 
     enum class MigrationState {
-        NotStarted,
-        Completed,
+        NotMigrated,
+        Migrated,
     };
 
     static MigrationStoreRef create(DBRef db,
                                     std::function<void(MigrationStore::MigrationState)>&& on_migration_state_changed);
 
-    // Converts the configuration from PBS to FLX if in the migrated state, otherwise returns the passed in config
-    // object. If the provided config is configured for FLX, the migration will be canceled and the migration state
-    // will be cleared.
-    std::shared_ptr<realm::SyncConfig> convert_sync_config(std::shared_ptr<realm::SyncConfig> config);
-
     // Called when the server responds with migrate to FLX and stores the FLX
     // subscription RQL query string
-    void migrate_to_flx(std::string rql_query_string);
+    void migrate_to_flx(std::string_view rql_query_string);
 
     // Clear the migrated state
     void cancel_migration();
 
+    // Clear the migration store info
+    void clear();
+
+    bool is_migrated();
+
+    std::string_view get_query_string();
```

Comment:
does not seem to be used

Reply:
These functions are for a forthcoming test of the migration store